Patch by Robert Scheck <robert@fedoraproject.org> for dsniff >= 2.4b1, that
adds some missing linkages to glib2.

--- dsniff-2.4b1/Makefile.in		2007-11-24 13:56:47.000000000 +0100
+++ dsniff-2.4b1/Makefile.in.glib2	2007-11-24 15:40:55.000000000 +0100
@@ -26,7 +26,7 @@
 LNETLIB = @LNETLIB@
 
 NIDSINC	= @NIDSINC@
-NIDSLIB	= @NIDSLIB@
+NIDSLIB	= @NIDSLIB@ -lglib-2.0 -lgthread-2.0 -lpthread
 
 DBINC	= @DBINC@
 DBLIB	= @DBLIB@
